content/strategy-goals/strategy/cloud/iam-admin/index.md

```diff
@@ -1,5 +1,47 @@
 # IAM and Admin Experience Team
 
-## Current and Future Goals
+This page outlines the short-term strategy for the IAM and Admin Experience Team. You can find the overview of Sourcegraph Product and Engineering FY23 Q2 focus [here](../../../../departments/product-engineering/strategy-goals/index.md#fy23-q2-focus)
 
-This team is newly formed and is currently defining goals.
+## Q2 Objective and Key Results
+
+### Problem Area 1
```

Comment:
Using numbers here makes is feel like they are in priority order, which I don't think is the intent.  Could we name the problem areas or maybe just make a bulleted list so we don't infer priority?

Reply:
Agreed - I removed the number and replaced with a short title of the problem area. How does this feel?